### ISSUE TITLE
[Request] Show which parts are missing in Mech Chassis construction.

### ISSUE THREAD

**Issue by 9600bauds** [d3athrow/vgstation13#3383]
It's not terribly obvious, at least not until you do it a thousand times over, if that chassis has a torso on it or not or if that missing leg is your left or the mech's left. It can lead to frustration and trial and error by rebuilding mech parts when you don't know which part out of them all is missing and none of your tools do anything.
Given that examining a machine frame tells you what components are missing it'd be nice to have this functionality in mech chassis too.

**Comment by sonime** [comment]
Agreed. Knowing which step is next would be very useful, especially for the Firefighter Ripley because you have to add a firesuit and gas mask at certain times that aren't covered under the wiki.
